### ISSUE TITLE
Routing Event Within Md-Menu-Item Doesn't Work in Angular 2 App

### ISSUE THREAD

**Issue by darrenbrett** [angular/material#10454]
My environment:
MacBook Air | OS X El Capitan Version 10.11.6
Browser: Chrome Version 56.0.2924.87 (64-bit)
@angular/cli: 1.0.0-beta.31
Node: 7.4.0

I am using material2 elements in various places in my Angular 2 app, but am running into an issue when it comes to using a routing event within an md-menu item. This is what my code looks like:

    <div class="user-menu">
        <button md-icon-button [mdMenuTriggerFor]="menu">
          <md-icon>more_vert</md-icon>
        </button>
        <md-menu #menu="mdMenu">
          <button md-menu-item>
            <md-icon>lock_outline</md-icon>
            <span [routerLink]="['/login']">Logout</span>
          </button>
          <button md-menu-item>
            <md-icon>voicemail</md-icon>
            <span>Check voicemail</span>
          </button>
        </md-menu>
    </div>

The problem is with this button element specifically:

      <button md-menu-item>
        <md-icon>lock_outline</md-icon>
        <span [routerLink]="['/login']">Logout</span>
      </button>
    
Simply put, the routing event doesn't work. The button flashes when you click it, but it doesn't route anywhere. 

Any idea why this would be an issue? 

Strangley, if I change the code to use the disabled menu-item, like this:

      <button md-menu-item disabled>
        <md-icon>lock_outline</md-icon>
        <span [routerLink]="['/login']">Logout</span>
      </button>

... the routing event works, but only when clicking on the text, not on the icon.
